packages/dap/src/client.spec.ts

```diff
@@ -367,8 +366,10 @@ describe("DAPClient", () => {
 
   describe("sending reports", () => {
     it("can succeed", async () => {
+      const params = buildParams();
+      const taskId = params.taskId.buffer.toString("base64url");
       const fetch = mockFetch({
-        "https://a.example.com/v1/upload": [{ status: 200 }],
+        [`https://a.example.com/v1/tasks/${taskId}/reports`]: [{ status: 201 }],
```

Comment:
I could use some help here: this test fails with:

```

  1) DAPClient
       sending reports
         can succeed:
     TypeError: Cannot read properties of undefined (reading 'shift')
      at DAPClient.fakeFetch (src/client.spec.ts:33:35)
      at DAPClient.sendReport (src/client.ts:31:294)
      at Context.<anonymous> (src/client.spec.ts:377:20)
```

But the same thing works just fine in the `"sending measurement"` test down below, so I don't understand why it fails here.

Reply:
added in 0b1884d, feel free to rebase away

Reply:
As a separate PR I'll improve the fetch mock error message

Reply:
Ooooh, that's embarrassing. Thank you!

Reply:
error message improved in #200